dpkgfiles/make_deb_package.README

```diff
@@ -0,0 +1,48 @@
+Instructions for using make_deb_package.py outside of Gitian
+
+
+1) We use cowbuilder to build in a chroot, so you need to prepare a chroot with
+   cowbuilder first (instructions from the Debian reproducible build project):
+
+      sudo cowbuilder --create --distribution codename --basepath /var/cache/pbuilder/base-reproducible.cow
```

Comment:
I can confirm that it doesn't break anything on Debian and I think is actually necessary. I thought I had put it in there from when Alan ran through the directions, but I guess not.